js/src/modals/CreateAccount/NewGeth/newGeth.js

```diff
@@ -106,7 +106,7 @@ export default class NewGeth extends Component {
     api.parity
       .listGethAccounts()
       .then((_addresses) => {
-        const addresses = (addresses || []).filter((address) => !accounts[address]);
```

Comment:
Interesting, this should have been catched by the linter with `_addresses` being an unused variable.